src/meili/routing.cc

```diff
@@ -506,12 +506,12 @@ find_shortest_path(baldr::GraphReader& reader,
     } else {
```

Comment:
if the changes you want to make are only to rename variables, please please, put a comment in the PR that notes as much. with a large change set like this, its important to bring the reviewers attention to the correct places in the code. i understand that you renamed these variables for the sake of making the code easier to understand, no problem there, but its quite distracting to spend time reviewing changes that have no impact on the code. please in the future, when you do this sort of thing, mention it up front.